File: app_delivery_phase_1_writing_code/golang_hello/hello.go
package main

import "fmt"

func main() {
	var input string

	fmt.Print("Will you subscribe to the devhulk YouTube channel? (Yes or No) \n")
	fmt.Scan(&input)
	if input == "Yes" {
		fmt.Println("Thank you!")
	} else {
		fmt.Println(":(")
	}
}

File: app_delivery_phase_1_writing_code/javascript_hello/hello.js
const readUserInput = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});

readUserInput.question('Will you subscribe to the devhulk YouTube channel? \n', answer => {

    if (answer == 'Yes') {
      console.log('\n Thank you!');

    } else if (answer == 'No') {
      console.log('\n :(');

    }
      readUserInput.close();
});

